### ISSUE TITLE
 List how many cards for revision (2016-04-12)

### ISSUE THREAD

**Issue by drbeco** [drbeco/qualcard#6]
Before starting, tell how many cards are due to today

**Comment by drbeco** [comment]
Issue #12 was marked as a duplicate of this issue.  2016-04-15

**Comment by drbeco** [comment]
changed status to resolved
Done on summary (-s) for yesterday
  2016-04-18
